config/samples/vpcresourcerequest.yaml

```
@@ -0,0 +1,18 @@
+apiVersion: vpcresources.k8s.aws/v1beta1
```

Consider starting the API as an alpha.
```suggestion
apiVersion: vpcresources.k8s.aws/v1alpha1
```